Backend/Planer_zakupowy.Backend.Api/Program.cs

```diff
@@ -16,8 +16,21 @@
 
 var builder = WebApplication.CreateBuilder(args);
 var connection = builder.Configuration.GetConnectionString("DBConnection");
+var allowSpecificOrigins = "allowSpecificOrigins";
 
 builder.Services.AddControllers();
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy(name: allowSpecificOrigins,
+                      policy =>
+                      {
+                          policy
+                          .WithOrigins("http://localhost:3000",
```

Comment:
It won't work on azure instance ;)

Reply:
Yes, but now we have frontend locally and only backend is deployed on Azure. Once the frontend is  deployed, this policy will be changed.